model-server/src/main/kotlin/org/modelix/model/server/handlers/ModelReplicationServer.kt

```diff
@@ -240,11 +240,11 @@ class ModelReplicationServer(
     override suspend fun PipelineContext<Unit, ApplicationCall>.postRepositoryObjectsGetAll(repository: String) {
         checkPermission(ModelServerPermissionSchema.repository(repository).objects.read)
         runWithRepository(repository) {
-            val keys = call.receiveStream().bufferedReader().use { reader ->
-                reader.lineSequence().toHashSet()
-            }
-            val objects = withContext(Dispatchers.IO) { modelClient.store.getAll(keys) }.checkValuesNotNull {
-                "Object not found: $it"
+            val objects = withContext(Dispatchers.IO) {
+                val keys = call.receiveStream().bufferedReader().use { reader ->
```

Comment:
Looking back at the ktor documentation: why don't we use `receiveChannel` as a non-blocking version?

Reply:
I found no reason why it was not used.
I have the other occurrence for consistency too.